lib/ios/BackpackReactNative/Classes/SnackbarBridge/RCTBPKSnakbarManager.h

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
You misspelled 'snackbar' in this file name (also in the `.m` file)

Reply:
Damnit